keyboards/tmo50/tmo50.c

```diff
@@ -51,34 +51,51 @@ void led_set_kb(uint8_t usb_led) {
 	led_set_user(usb_led);
 }
 
-uint32_t layer_state_set_user(uint32_t state)
+uint32_t layer_state_set_kb(uint32_t state)
```

Comment:
Would prefer that these keyboard changes were in a separate PR. 

Also, 
```suggestion
layer_state_t layer_state_set_kb(layer_state_t state)
```

Reply:
Done. #10150